src/components/compound/DlSelect/components/DlSelectOption.vue

```diff
@@ -4,7 +4,17 @@
         :class="identifierClass"
         style="width: 100%"
     >
+        <div
+            v-if="readonly"
+            :class="[{ 'readonly-option': true }, { capitalized }]"
+            :style="`padding-left: ${10 + depth * 30}px;`"
+        >
+            <slot>
```

Comment:
this slot will be default name, is that fine ?

Reply:
In my opinion, it's fine unless you suggest otherwise.